### ISSUE TITLE
feat: add support for parameters in client.beta.threads.messages.list

### ISSUE THREAD

**Issue by pietroMonta42** [mharrisb1/openai-responses-python#32]
I have tried to add parameters, but it doesnt work and send:
Error:
```
Connection error.
respx.models.SideEffectError: <Route <Scheme eq 'https'> AND <Host eq 'api.openai.com'> AND <Method eq 'GET'> AND <URL regex re.compile('/v1/threads/(?P<thread_id>[a-zA-Z0-9\\_]+)/messages')>>

The above exception was the direct cause of the following exception:

TypeError: can only concatenate str (not "int") to str

The above exception was the direct cause of the following exception:
    client.beta.threads.messages.list(thread_id=thread_id, limit=limit, order=order, after=after, before=before)
  ```
  
  without the parameters the function works

**Comment by mharrisb1** [comment]
Thanks @pietroMonta42 

I'll need to dig a little bit more on this one but should be relatively straightforward

**Comment by mharrisb1** [comment]
@pietroMonta42 new release just published to PyPI with fixes for this and #31

**Comment by pietroMonta42** [comment]
wow, it's my first issue on GitHub, how fast!
Thank you very much
